src/System.Management.Automation/engine/Utils.cs

```diff
@@ -324,11 +285,7 @@ private static string[] GetProductFolderDirectories()
                 // TODO: #1184 will resolve this work-around
                 // Side-by-side versions of PowerShell use modules from their application base, not
                 // the system installation path.
-#if CORECLR
                 progFileDir = Path.Combine(appBase, "Modules");
-#else
-                progFileDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles), "WindowsPowerShell", "Modules");
```

Comment:
We'll have a different solution for supporting Windows PowerShell `PSModulePath` in PSCore6 that replaces the current interim solution